vitest.config.ts

```diff
@@ -0,0 +1,93 @@
+import { resolve } from 'node:path';
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+	resolve: {
+		alias: [
```

Comment:
Is this actually needed? I don't need it for dragonite for example

Reply:
It somehow didn't work at all without, it's a first for me too, also incredibly annoying, but I'm out of ideas.

Reply:
Tried removing it, it didn't work because it'll try to use the built JS files rather than the source files.
![image](https://github.com/skyra-project/skyra/assets/24852502/f95b6ed1-45da-41f5-b919-
d295ef3168f7)

*I mean, if we require building before testing, we solve this issue.*

Reply:
nah lets keep aliases then